### DISCUSSION TITLE
How can I remove some keybinding I don't want in lspconfig?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [LazyVim/LazyVim#1288] [Q&A] [answered]
I want use `gd` for `Lspsaga goto_definition`, but I found I haven't delete this line in `Lazyvim` to make my keybinding take effect.

<img width="1789" alt="image" src="https://github.com/LazyVim/LazyVim/assets/95092244/bf0950b9-8596-45ae-9e6e-021acbc53914">

**Comment by dpetka2001** [comment, marked as answer]
[Docs](https://www.lazyvim.org/plugins/lsp#%EF%B8%8F-customizing-lsp-keymaps) have the answer.

**Reply by LintaoAmons** [reply]
Thx！

We should have a link under Keymap page LSP section

**Reply by dpetka2001** [reply]
If you press on Lsp link  it will take you there. On the right column you can see the subsections of the page and it mentions `Customizing LSP keymaps`.

**Reply by LintaoAmons** [reply]
https://www.lazyvim.org/keymaps#lsp

I was trying to find it here....
